src/main/java/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarm.java

```diff
@@ -488,7 +488,31 @@ public ListSuitesResult listSuites(String runArn) {
         return api.listSuites(request);
     }
 
+    /*
+     * get number of Unmetered devices based on upload type
+     */
+    public int getUnmeteredDevices(String os) {
```

Comment:
I have removed it for now. It make it consistent with rest of methods in the file